Onboarding/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,38 @@
+name: Continuous integration
```

Comment:
This should be in `.github/workflows` in the repo root, not under `Onboarding`.

Reply:
Thanks